backend/src/index.ts

```diff
@@ -1,19 +1,19 @@
 import express from 'express';
-import mongoose from 'mongoose';
+import '../src/db/mongoose';
 
 const app = express();
 const PORT = process.env.PORT || 8000;
-const dbLink = 'mongodb://localhost:27018/db';
 
-mongoose.connect(dbLink, err => {
-  if (err) {
-    console.error('Error occured during connection to DB');
-    console.error(err);
-  } else {
-    console.log('Connected to DB succesfully');
-  }
+import routes from './routes/index';
+
+app.all('*', function (req, res, next) {
+  res.header('Access-Control-Allow-Origin', '*');
```

Comment:
I wonder if we plan to do some cross site requests? 
This setting is quite dangerous in terms of security